logos/amplify/amplify.svg

```diff
@@ -0,0 +1,95 @@
+     id="namedview4"
+     showgrid="false"
+     inkscape:zoom="2.2780556"
+     inkscape:cx="112.5"
+     inkscape:cy="112.5"
+     inkscape:window-x="-8"
+     inkscape:window-y="-8"
+     inkscape:window-maximized="1"
+     inkscape:current-layer="svg2" />
+  <image
+     width="225"
+     height="225"
+     preserveAspectRatio="none"
+     xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAOEAAADhCAMAAAAJbSJIAAAAmVBMVEXZG1z////hV4LYFFndN27Y
+DVfld5fXAFHYAFTXAE799vbzx83ZFVXlepDXAFDXAE3sprHdRG7pkaHrnrHur7n77vH55OXfVXf4
+4OXeSm321tnfVHvroK7bMF/ql6bwucbxvsX0zdLkcov88vLgW3v3297mhJfurbjiaIPcPGneR2va
+JlrfUXLsorPoi57kdYzlgJvVAEHqkq3pZgULAAAGX0lEQVR4nO2dW3uqOBRAYXIyCV5AraJ1xIK0
+Vapjz5n//+MmF6hIgkaPfp3t7PUiQm6LkGQTH/T+IN5D88NDQ+igIXzQED5oCB80hA8awgcN4YOG
+8EFD+KAhfNAQPmgIHzSEDxrCBw3hg4bwQUP4oCF80BA+aAgfNIQPGsIHDeGDhvBBQ/igIXzQED5o
+CB80hA8awgcN4YOG8EFD+KAhfNAQPmgIHzSEDxrCBw3hg4bwQUP4oCF80BA+aAgfNLwSyhPC71Du
+FdzHkG9iP1+y2xd8BXcxJENfsvxP9OJdDPlYGU7Cm5d8BXcxZF1l2OndvOQrQMOreGRDXdbjGrLt
+R0C9BzakqbBa8cc1ZHqNKAhkQ8JCRtsuMqXlL3jdkNAGpKpOfjHKUCmqysq8+gu1pLeebHKRIR2m
+826/LRjracNB3ZA89xu8flAma0zkl6RZwarff5/Ky4Tx4vVTplnJ+shKZd4e6fCpPPd+Q8MyGBu0
+hCo2Q770TbIh9chUHj036o5y339TVSVpp0qeijtCpurrpKgpBu86NjzTiZcYspmucmov02o4shiK
+EkiL4cTvhlQK5rXUbz3i0SRWRU6/Yt1woS6uzsX3Fxh+tTa2d+IFhm9Bi2EnjQrRKSw9Sj4Zil4k
+c3W80ZWTSCd5PRvduxuS4qvGo5cGwpKEnTTsPB34WzdsS62GdBk9+2PucfVMvg2qTHvZr5EudS8V
+S9+8OP/64m4YqAp26vFJDnnIcO53UnbSMOIHor2+9VZDLxDjTRgm8mJ6yKaHRaiXo0VUPbOT5NxE
+eokh/ZRlrqOV/MiCw3l1uzN2yvCoy9WjsLEb0o04OebkVd0Fo/mRHnpptFOVzrlL050Niey83COB
+ejxWVe20r73EOn8DQzHTKEN1cWg2LFQPgD/XdzV0armrYajGzycvW5hXmfiT9vrr9obPrHeMjAc3
+X5PBOHJpt7OhnhgyOciZelTG4ZWGfOdq2Gngj0Sd/LVcJweB54ajIYsPzw1Xxzvqbhgc+iGZvik1
++1x6ZGgiu40Waq7bO/agq6Hut+UvPRuqeWDGnQ0ttK2Hpw39VNzW8EXW7izoZkjU7O3HJTq0GfHr
+DRfcwTA2mYxFNCANY/edSifDMLM1Uy2K1xh2FqyMSz9OGRYBMxB9eA9D3rd2RDd0NVzU2W+2ovdJ
+OeGcMNzZG3aXPpTTV2dcY60mG7koOs009ZiGly90W/UgFlF4gH2XoRra/rLezFBNNnnvitWiQsmI
+ps4PZMn3GFJlEx9PXjpEHLPrDfneHJ/fZKiXwmaQyNTCu6NXG3r8+A3p+wz5Wtb20nwl1IH4xGK4
+FoaZPqze8e2vOMEqjTt+nSSS31/KyLvNUN6ZyU3HIRFY9ozk6epy/YgcHXnVpwXKGKekRpnxdK6v
+NI6cN6y11nKpNb3lkvVUe/KW8ltPt3DecKgomOVRK4ZFMzNhxbDMYTTMcm54wCidJEOzfHm+MMs5
+wVnDpBwj+cJ49MWIyBonSRFXg2rdvCVhLGeho9TD2iA0hl0kisot4y2I/e4FPy+7GD79/LkXE1i3
+2WRp2JiB6LuIYJaKTXPsBoahl4h0ozc/Hi2XC2PvVEVSlt+Ro9sbThnlkajvtZHSYihWhw6VIUEY
+Gk2zGHryVUWUIqOJZnLxktZd+7k5x0Wzmxsqs17u75trot1QfGw7+SSfNgq2GdpLkcgufE5snXgv
+Q3aRYdaV277H1y4zFKnjzWZu+XnnToZUfL47G3rh9jcN6df7jNGJdzDcRVGYZEe7pG1to8KQRYLE
+HLYXGYpYMfvY7T5SM+YThumvSHGT3UTR1IlA3MzP5qAXbZs3DVc6uczQ3A1sM+xaDGUXTkW8Qwuz
+E8Vc2tF1TGzLZZPzhvFMki0Sc7YbzF6MyXGUqfSzbt9YXLqzkWWPWpQyNooWaXUozAazrDk4urOS
++CaGXqAIbfvLPDD7hIc6g+WXVBZYN+FtpYi0rLpqRhpBxU33vMGChvBBQ/igIXzQED5oCB80hA8a
+wgcN4YOG8EFD+KAhfNAQPmgIHzSEDxrCBw3hg4bwQUP4oCF80BA+aAgfNIQPGsIHDeGDhvBBQ/ig
+IXzQED5oCB80hA8awgcN4YOG8EFD+KAhfNAQPmgIHzSEDxrCBw3hg4bwQUP4oCF8/heGvPm3Wo/F
+D++fPx+bn/8CeIODmnCqdDIAAAAASUVORK5CYII=
+"
+     id="image10"
+     x="0"
+     y="0" />
```

Comment:
Also, your SVGs should contain pure vectors. 
You have uploaded pngs and wrapped as SVGs, as you can see from the `<image` tag highlighted. This isn't pure vector and as a result, you can't edit Colours and paths in vector tools like illustrator, etc.

Please replace your logos with vector

Reply:
Okay I'm using inkscape here so I saved it as SVG, what other vector tool would you suggest I use?